azure/packages/test/end-to-end-tests/package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"name": "@fluidframework/azure-end-to-end-tests",
 	"version": "2.0.0-internal.8.0.0",
+	"private": true,
```

Comment:
this might break our e2e test pipeline, but i'm not sure

Reply:
I didn't see any references to this package name anywhere else in the repo, FWIW.

Reply:
Ah, so our pipelines are installing these from our internal feed then? If so, should these move to the `fluid-private` namespace?

Reply:
i don't have any of those answers, i'm just vaguely aware that this could break something

Reply:
In general, I would be wary of ceasing to publish any package that's on the feeds list. The e2e tests definitely need to be published to one of the internal feeds. I _think_ the fluid build package config has comments describing why some of them are published - if it doesn't then maybe we can stop and see what happens. :)

Reply:
I think this is all correct, but also a pattern we should move away from (i.e. installing a version of the e2e tests package published to our feeds in order to run them, because that won't respect the lockfile at the root of the release group). We have the same issue when running perf tests today. Yes, we might pay for that change in pipeline execution time but seems like the correct choice anyway.